contracts/token-distribution/SaleRounds.sol

```diff
@@ -151,6 +151,29 @@ contract SaleRounds is TokenDistribution, GameOwner, ERC20 {
         vestingStartTime = block.timestamp;
     }
 
+    function editReservedAllocations(string calldata _roundType, address _to, uint _amount) external
+    onlyGameOwner {
+        require(vestingStartTime > block.timestamp, "Token vesting has begun.");
+        RoundType roundType = getRoundTypeByKey(_roundType);
+        require(roundDistribution[roundType].supply >= _amount, "given amount is bigger than max supply for the round");
+        require(reservedBalances[roundType][_to] > 0, "there is no reserved balance");
+        require(roundDistribution[roundType].totalRemaining > 0, "there is no remaining round distribution amount");
```

Comment:
This could break if the all token is allocated, but amount is a smaller new amount.